internal/switcher/frr.go

```diff
@@ -42,7 +42,7 @@ func NewFrrApplier(c *Conf) Applier {
 func (a FrrApplier) Apply() error {
 	tpl := a.getTpl()
 	ok, err := a.applier.Apply(*tpl, FrrTmp, Frr, true)
-	if !ok {
+	if !ok && err != nil {
```

Comment:
I don't really understand why there are two values returned by this function, which more or less mean the same thing? I think this theoretically implies weird combinations which we don't catch properly here. Maybe just check the error and swallow the ok?

Reply:
OK

Reply:
done